vue-components/src/components/Lookup.vue

```diff
@@ -137,6 +140,20 @@ export default Vue.extend( {
 			// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
 			this.$el.querySelector( 'input' )!.blur();
 		},
+		onScroll( firstIndex: number, lastIndex: number ): void {
+			/**
+			 * This event is emitted whenever the first or last index of the
+			 * visible menuItems changes. If the user scrolls but the indexes remain
+			 * unchanged the event won't fire.
+			 *
+			 */
+			if ( firstIndex !== this.scrollIndexStart || lastIndex !== this.scrollIndexEnd ) {
+				this.$emit( 'scroll', firstIndex, lastIndex );
```

Comment:
Ooops, I approved too soon, I think this needs to be right above the `$emit` to appear in the docs.

![Screenshot from 2020-11-09 17-04-46](https://user-images.githubusercontent.com/6132917/98565374-d600df00-22ad-11eb-9b99-cc63d53a7410.png)

```suggestion
			if ( firstIndex !== this.scrollIndexStart || lastIndex !== this.scrollIndexEnd ) {
			      /**
			       * This event is emitted whenever the first or last index of the
			       * visible menuItems changes. If the user scrolls but the indexes remain
			       * unchanged the event won't fire.
			       *
			       */
				this.$emit( 'scroll', firstIndex, lastIndex );
```